<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWidget</class>
 <widget class="QWidget" name="GroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mGroupLabelBK">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>629</width>
     <height>67</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mPButtonGroupSpace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelGroupNickChat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>8</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="mGroupStackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>111</y>
     <width>629</width>
     <height>489</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="mTButtonGroupFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
